postal_address/tests/test_territory.py

```diff
@@ -247,3 +257,42 @@ def test_subdivision_type_id_collision(self):
                     self.assertTrue(hasattr(simple_address, metadata_id))
                 else:
                     self.assertFalse(hasattr(simple_address, metadata_id))
+
+    def test_subdivision_parent_code(self):
+        # See https://bitbucket.org/flyingcircus/pycountry/issues/13389
+        self.assertEqual("GB-ENG", subdivisions.get(code='GB-STS').parent_code)
+        self.assertEqual("CZ-20", subdivisions.get(code='CZ-205').parent_code)
+
+    def test_foreign_territory_mapping(self):
+        self.assertEqual("FR", territory_attachment("GP"))
+        self.assertEqual("NL", territory_attachment("BQ"))
+
+    def test_normalize_territory_code(self):
+        self.assertEqual("GR", normalize_territory_code("EL"))
+        self.assertEqual("FR", normalize_territory_code("FX"))
+        self.assertEqual("TW", normalize_territory_code("CN-71"))
+        # Sub-territories will not change by default
+        self.assertEqual("BQ", normalize_territory_code("BQ"))
+        self.assertEqual("GP", normalize_territory_code("FR-GP"))
+        # TODO: Is it normal to not retrieve alpha2 ?
```

Comment:
I guess this is allowed as the concept of a "territory" is covering both county and subdivisions. So to me, based on its naming `normalize_territory_code()` should return the most appropriate territory, be it a country or a subdivision.